.github/workflows/python-docs.yml

```diff
@@ -0,0 +1,20 @@
+name: "Pull Request Docs Check"
+on: 
+- pull_request
+- push
+
+jobs:
+  docs:
+
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Maybe we could run this with only one python version, since docs shouldn't really depend on the python version. Is there a `latest` version?

Reply:
i don't the option to use python latest version, so i choose 3.8. there is also `3.x` option but it is not clear to me if these mean single version or all possible python 3 version.